nc/source/engine/Engine.cpp

```diff
@@ -33,6 +33,20 @@ namespace
             .worldspaceExtent = config.physics.worldspaceExtent
         };
     }
+
+    #ifdef NC_EDITOR_ENABLED
```

Comment:
Can you explain why this is only here for the editor? I'm missing something here